Mobile/ShopLens/Droid/CameraActivity.cs

```diff
@@ -22,13 +22,17 @@ public class CameraActivity : Activity
     {
         Button BtnTakeImg;
         ImageView ImgView;
+        Button BtnPickImg;
+        public static readonly int PickImageId = 1000;
```

Comment:
I don't know whether this field needs to be static. Is "static readonly" a conventional thing for activity Id's or will we have another class that will use this?

Reply:
I believe it should be static readonly because it's only usage is for request code, neither we need it to be writable neither we need a new instance for every object.